yt/utilities/command_line.py

```diff
@@ -1159,6 +1179,32 @@ def __call__(self, args):
             pprint.pprint(rv)
 
 
+class YTUploadFileCmd(YTCommand):
+    args = (dict(short="file", type=str),)
+    description = \
+        """
+        Upload a file to yt's curldrop.
+
+        """
+    name = "upload"
+
+    def __call__(self, args):
+        try:
+            import requests
+        except ImportError:
+            print("yt {} requires requests to be installed".format(self.name))
+            print("Please install them using your python package manager, e.g.:")
+            print("   pip install requests --user")
```

Comment:
I would say "please install requests with the package manager appropriate for your python environment" to accomodate for conda. Also `--user` will not be appropriate for some python environments.